docs/admin-api-brokers.md

```diff
@@ -87,39 +69,53 @@ Fetch the information of the leader broker, for example, the service url.
 <TabItem value="pulsar-admin">
 
 ```shell
-pulsar-admin brokers leader-broker
+pulsar-admin brokers namespaces use \
+    --url localhost:8080
 ```
 
 Example output:
 
-```json
-{
-  "serviceUrl" : "http://localhost:8080"
-}
+```
+public/default/0x00000000_0x40000000    [broker_assignment=shared is_controlled=false is_active=true]
+public/default/0xc0000000_0xffffffff    [broker_assignment=shared is_controlled=false is_active=true]
+public/functions/0x40000000_0x80000000    [broker_assignment=shared is_controlled=false is_active=true]
+public/functions/0x00000000_0x40000000    [broker_assignment=shared is_controlled=false is_active=true]
+pulsar/standalone/localhost:8080/0x00000000_0xffffffff    [broker_assignment=shared is_controlled=false is_active=true]
+pulsar/localhost:8080/0x00000000_0xffffffff    [broker_assignment=shared is_controlled=false is_active=true]
+public/functions/0x80000000_0xc0000000    [broker_assignment=shared is_controlled=false is_active=true]
+public/default/0x80000000_0xc0000000    [broker_assignment=shared is_controlled=false is_active=true]
 ```
 
 </TabItem>
 <TabItem value="REST API">
 
-{@inject: endpoint|GET|/admin/v2/brokers/leaderBroker|operation/getLeaderBroker?version=@pulsar:version_number@}
+{@inject: endpoint|GET|/admin/v2/brokers/:cluster/:broker/ownedNamespaces|operation/getOwnedNamespaes?version=@pulsar:version_number@}
 
 </TabItem>
 <TabItem value="Java">
 
 ```java
-admin.brokers().getLeaderBroker()
+admin.brokers().getOwnedNamespaces(cluster,brokerUrl);
 ```
 
-For the detail of the code above, see [here](https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java#L80)
-
 </TabItem>
 
 </Tabs>
 ````
 
-#### list of namespaces owned by a given broker
+## Update broker conf 
+
+You can update broker configurations using one of the following ways:
 
-It finds all namespaces which are owned and served by a given broker.
+- Supply [configurations](https://pulsar.apache.org/reference/#/@pulsar:version_origin@/config/reference-configuration-broker) when starting up brokers.
+
+- [Update configurations dynamically](#update-broker-conf-dynamically) **when running brokers**.
+
+  Since all broker configurations in Pulsar are stored in ZooKeeper, configuration values can also be dynamically updated when brokers are running. When you update broker configurations dynamically, ZooKeeper will notify the broker of the change and then the broker will override any existing configuration values.
+
+### List updatable broker conf
```

Comment:
It seems more like an optional step before users dynamically update broker conf. The same applies to `List updated broker conf`. Logically, they are talking about the `list` feature, not `update`, but can be leveraged before/after updating broker conf.

So I suggest a tweak on heading 2 and heading 3, using either of the following. 

**OptionA:**
```
## List updatable broker conf
...
## List updated broker conf
...
## Update broker conf dynamically
... // link to the the List headings
```
**OptionB:**
```
## Update broker conf dynamically
1. List updatable broker conf
2. ...
3. List updated broker conf
```

The difference between A and B is whether we want to highlight the feature of `list`. WDYT?

Reply:
I'd prefer not to use an ordered list here. Reasons:

1. This page is a "reference" topic rather than a "task" topic.
  
    In a "task" topic, ordered lists in headings make sense because the sequence matters.

<img width="1960" alt="image" src="https://user-images.githubusercontent.com/50226895/228206657-001cb444-40c2-4e35-a5d6-a40aaef6364c.png">
    
   But in a "reference" topic, it's rare to use ordered lists for operations because sequence does not matter most. 

2. `List updatable broker conf`, `Update broker conf dynamically`, and `List updated broker conf` are not complex operations. Users can know their logic at 1st glance.

Reply:
It's not a big issue, but I wanted to make myself clear for your reference :)
The possible confusion exists because the relationship between heading 2 and nested three heading 3 is not self-explained, making readers to think and guess (I did)... e.g.,
* Are those heading3 separate sub-features? Why 'list' appears here?
* If a `dynamic` item is a sub-feature, why a contrary one is absent? It's not MECE.
* What's the relationship amount the three? Do they all serve the heading2 `Update broker conf`? After reading the content, I found the `list` ones serve `Update broker conf dynamically`.

<img width="241" alt="image" src="https://user-images.githubusercontent.com/60642177/228223638-3bece14b-f690-4121-941d-57c7a9a63e70.png">

That's why I suggest changing the heading3 into plain-text ordered steps. BTW, I don't see this category as a reference since we have a doc set including overview and get-started topics for it. It's more like a key feature. The topic title "Brokers" means "Manage brokers" in my eyes.

Reply:
> Are those heading3 separate sub-features? Why 'list' appears here?

Yes, they are all sub-features.

> If a dynamic item is a sub-feature, why a contrary one is absent? It's not MECE.

Because we do not have the contrary feature.

> What's the relationship amount the three? Do they all serve the heading2 Update broker conf? After reading the content, I found the list ones serve Update broker conf dynamically.

They all work for heading2 `update broker conf` because e only have `update broker dynamically` and do not have the contrary one. So`update broker conf` is an appropriate heading2 for all 3 operations.

> I don't see this category as a reference since we have a doc set including overview and get-started topics for it.

Don't be puzzled by the title, location, and doc structure. It looks like a "concept" or "get started" topic at 1st glance. 

But if taking a closer and careful look, you'll find that the docs are talking about "command references" rather than "what is it" or "how to use it". So it's a "reference" topic in essence.